scene/2d/tile_map.cpp

```diff
@@ -346,7 +346,10 @@ void TileMap::update_dirty_quadrants() {
 	Color debug_collision_color;
 	Color debug_navigation_color;
 
-	bool debug_shapes = show_collision && (Engine::get_singleton()->is_editor_hint() || (st && st->is_debugging_collisions_hint()));
+	bool debug_shapes = false;
+#ifdef DEBUG_ENEBLED
+	debug_shapes = show_collision && (Engine::get_singleton()->is_editor_hint() || (st && st->is_debugging_collisions_hint()));
+#endif
```

Comment:
There's a typo: `DEBUG_ENEBLED` instead of `DEBUG_ENABLED` :)

But this check is actually not needed in this version, since `is_debugging_collisions_hint()` will always return false in release, and we do want collision in the editor in all configurations. It was only needed when we had an option to set it always on.

Reply:
oh damn. my bad. :disappointed:

Reply:
Fixed.